docs/userguide/networking/get-started-overlay.md

```diff
@@ -11,6 +11,36 @@ weight=-3
 
 # Get started with multi-host networking
 
+As of Docker Engine 1.12 and later, you can create an overlay network on a manager node running in [swarm mode](../../swarm/swarm-mode.md).
+
+The swarm makes the overlay network available only to nodes in the swarm that
+require it for a service. When you create a service that uses the overlay
+network, the manager node automatically extends the overlay network to nodes
+that run service tasks.
+
+To learn more about running Docker Engine in swarm mode, refer to the
+[Swarm mode overview] (../../swarm/index.md).
+
+The example below shows how to create a network and use it for a service from a manager node in the swarm:
+
+```bash
+# Create an overlay network `my-multi-host-network`.
+$ docker network create --driver overlay my-multi-host-network
+
+400g6bwzd68jizzdx5pgyoe95
+
+# Create an nginx service and extend the my-multi-host-network to nodes where
+# the service's tasks run.
+$ $ docker service create --replicas 2 --network my-multi-host-network --name my-web nginx
+
+716thylsndqma81j6kkkb5aus
+```
+
+Overlay networks for a swarm are not available to unmanaged containers. For more information refer to [Note on Docker 1.12 Overlay Network Security Model](overlay-security-model.md).
```

Comment:
what is "managed / unmanaged" containers? I'm thinking you're talking about service tasks  containers vs adhoc ones, but without a definition used consistently, i'm feeling unsure.